### DISCUSSION TITLE
Deploy->Kubernetes->Yarn error

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Ed/Community#287] [Installation] [answered]
Hi Team,

CSP: GCP
I am trying to deploy a yarn it fails
Build version: release-2.6.0

`
TASK [Ensure yarn nodemanager is running] **************************************
fatal: [10.0.2.7]: FAILED! => {}

MSG:

Failed to set permissions on the temporary files Ansible needs to create when becoming an unprivileged user (rc: 1, err: chown: invalid user: ‘hduser’
}). For information on working around this, see [https://docs.ansible.com/ansible/become.html#becoming-an-unprivileged-user](https://docs.ansible.com/ansible/become.html#becoming-an-unprivileged-user%1B[0m)`

**Comment by beepdot** [comment, marked as answer]
Can you enable pipelining in ansible? Please search on google to understand how to enable ansible pipelining.

**Reply by tsprasath** [reply]
@keshavprasadms Added host entry in the private repo `core>hosts`

`[ssh_connection]`
`pipelining = True`

Yarn Deploy passed. Close this thread.
